distros/kubernetes/nvsentinel/templates/node-condition-cleanup-hook.yaml

```
@@ -0,0 +1,167 @@
+          {{- toYaml .Values.nodeConditionCleanup.resources | nindent 10 }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsNonRoot: true
+          runAsUser: 65534
+          capabilities:
+            drop:
+            - ALL
+          seccompProfile:
+            type: RuntimeDefault
+{{- end }}
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)